src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -274,6 +275,8 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 
 export interface ITerminalAction extends IAction {
 	source: string;
+	uri?: URI;
```

Comment:
I considered using some other property here instead of passing in either of these since they don't seem to get used elsewhere, but figured that would be more confusing